src/common/eventType.ts

```diff
@@ -35,15 +35,14 @@ export enum EventType {
 
     // Runtime dependency
     DependencyInstallStarted,
-    DependencyInstallFinished,
+    DependencyInstallCompleted,
     PackageInstallStarted,
-    PackageInstallSucceeded,
-    PackageInstallFailed,
+    PackageInstallCompleted,
+    PackageInstallError,

```

Comment:
yes, I have considered about this name, but I worried that it may be taken as an inheritor of `PackageInstallCompleted`.

---
In reply to: [382362276](https://github.com/docascode/vscode-docs-build/pull/61#discussion_r382362276) [](ancestors = 382362276)

Reply:
renamed to `PackageInstallAttemptFailed`

---
In reply to: [382403083](https://github.com/docascode/vscode-docs-build/pull/61#discussion_r382403083) [](ancestors = 382403083,382362276)